config.toml

```
@@ -57,12 +57,14 @@ rdi_db_types = "cassandra|mysql|oracle|postgresql|sqlserver"
 rdi_cli_latest = "latest"
 
 [params.clientsConfig]
-"Python"={lang="python", quickstartSlug="python/redis-py"}
-"Node.js"={lang="javascript", quickstartSlug="nodejs"}
-"Java"={lang="java", quickstartSlug="java/jedis"}
-"Go"={lang="go", quickstartSlug="go"}
-"C#"={lang="C#", quickstartSlug="dotnet"}
-"RedisVL"={lang="python", quickstartSlug="python/redis-vl"}
+"Python"={quickstartSlug="python/redis-py"}
+"Node.js"={quickstartSlug="nodejs"}
+"Java sync"={quickstartSlug="java/jedis"}
+"Java async"={quickstartSlug="java/lettuce"}
+"Java reactive"={quickstartSlug="java/lettuce"}
+"Go"={quickstartSlug="go"}
+"C#"={quickstartSlug="dotnet"}
+"RedisVL"={quickstartSlug="python/redis-vl"}
```

We merged a PR to flatten out these folders a couple of days ago, so we don't need the folders for Python and Java now:
```suggestion
"Python"={quickstartSlug="redis-py"}
"Node.js"={quickstartSlug="nodejs"}
"Java sync"={quickstartSlug="jedis"}
"Java async"={quickstartSlug="lettuce"}
"Java reactive"={quickstartSlug="lettuce"}
"Go"={quickstartSlug="go"}
"C#"={quickstartSlug="dotnet"}
"RedisVL"={quickstartSlug="redis-vl"}
```

---

@andy-stark-redis Are you sure? In both `main` and `latest` branches, those slugs are still unchanged:
https://github.com/redis/docs/blob/latest/config.toml#L59-L65
https://github.com/redis/docs/blob/main/config.toml#L59-L65

---

Yes, main and latest are currently wrong - I just didn't realise that until I saw this PR! (The quickstartSlug links still work, btw, but it happens with a redirect).